features/telemetry/adapters/netflow/src/main/java/org/opennms/netmgt/telemetry/adapters/netflow/AbstractAdapter.java

```diff
@@ -63,8 +65,11 @@
      */
     private final Histogram packetsPerLogHistogram;
 
-    public Netflow5Adapter(MetricRegistry metricRegistry, FlowRepository flowRepository) {
+    public AbstractAdapter(final MetricRegistry metricRegistry,
+                           final FlowRepository flowRepository,
+                           final Converter<P> converter) {
         this.flowRepository = Objects.requireNonNull(flowRepository);
+        this.converter = Objects.requireNonNull(converter);
 
         logParsingTimer = metricRegistry.timer("logParsing");
```

Comment:
The metrics from these objects are stored in attributes named `NF5PktsPerLog*`. Since we're reusing this code among adapters, can we change the `NF5` prefix to `Flow` instead? We'll need to update the JMX collection and graph definitions.

Reply:
Fixed